test/PriceOracleTest.js

```diff
@@ -0,0 +1,79 @@
+const { ethers, network } = require("hardhat");
+const { expect } = require("chai");
+
+const JXJOE_AGGREGATOR_ADDRESS = "0x02D35d3a8aC3e1626d3eE09A78Dd87286F5E8e3a";
```

Comment:
This is technically a price feed for JOE, so let's rename it to `JOE_AGGREGATOR_ADDRESS`